<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>323</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>General settings</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="OPTIONTABS">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="MOBIDICtab">
      <attribute name="title">
       <string>MOBIDIC</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QLineEdit" name="MATLAB"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="EXE_LBL">
         <property name="text">
          <string>Path to MOBIDIC</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="EXE_BTN">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="MATLAB_LBL">
         <property name="text">
          <string>Path to MATLAB</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="EXE"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="MATLAB_BTN">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layerTab">
      <attribute name="title">
       <string>Layers</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="LINKS_LBL">
         <property name="text">
          <string>Links</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="LINKS_BTN">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="NODES_LBL">
         <property name="text">
          <string>Nodes</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="NODES_BTN">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="WHEATHERSTATIONS_BTN">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QPushButton" name="LANDUSES_BTN">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QPushButton" name="SOILS_BTN">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QPushButton" name="ACQUIFERS_BTN">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>55</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="WHEATHERSTATIONS_LB">
         <property name="text">
          <string>Wheather</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="LANDUSES_LB">
         <property name="text">
          <string>Landuses</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="SOILS_LB">
         <property name="text">
          <string>Soils</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="ACQUIFERS_LB">
         <property name="text">
          <string>Acquifers</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QLineEdit" name="LINKS"/>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QLineEdit" name="NODES"/>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QLineEdit" name="WHEATHERSTATIONS"/>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="QLineEdit" name="LANDUSES"/>
       </item>
       <item row="4" column="1" colspan="3">
        <widget class="QLineEdit" name="SOILS"/>
       </item>
       <item row="5" column="1" colspan="3">
        <widget class="QLineEdit" name="ACQUIFERS"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advanceTab">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="DEBUGPATH_LBL">
         <property name="text">
          <string>debug folder:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="DEBUGPATH"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="DEBUGPATH_BTN">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="DEBUGMODE">
         <property name="text">
          <string>Debug mode</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>177</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
